tests/params/test_validator_bet_disks.py

```diff
@@ -0,0 +1,343 @@
+import copy
+import unittest
+
+import pydantic as pd
+import pytest
+
+import flow360 as fl
+from flow360.component.flow360_params.boundaries import (
+    FreestreamBoundary,
+    HeatFluxWall,
+    IsothermalWall,
+    MassInflow,
+    MassOutflow,
+    NoSlipWall,
+    RotationallyPeriodic,
+    SlidingInterfaceBoundary,
+    SlipWall,
+    SolidAdiabaticWall,
+    SolidIsothermalWall,
+    SubsonicInflow,
+    SubsonicOutflowMach,
+    SubsonicOutflowPressure,
+    SupersonicInflow,
+    TranslationallyPeriodic,
+    WallFunction,
+)
+from flow360.component.flow360_params.flow360_output import (
+    AeroacousticOutput,
+    IsoSurface,
+    IsoSurfaceOutput,
+    MonitorOutput,
+    SliceOutput,
+    SurfaceOutput,
+    VolumeOutput,
+)
+from flow360.component.flow360_params.flow360_params import (
+    BETDisk,
+    BETDiskChord,
+    BETDiskSectionalPolar,
+    BETDiskTwist,
+    Flow360Params,
+    FreestreamFromMach,
+    FreestreamFromVelocity,
+    MeshBoundary,
+    SteadyTimeStepping,
+)
+from flow360.component.flow360_params.initial_condition import (
+    ExpressionInitialCondition,
+)
+from flow360.component.flow360_params.solvers import (
+    HeatEquationSolver,
+    IncompressibleNavierStokesSolver,
+    SpalartAllmaras,
+    TransitionModelSolver,
+)
+from flow360.component.flow360_params.time_stepping import UnsteadyTimeStepping
+from flow360.component.flow360_params.turbulence_quantities import TurbulenceQuantities
+from flow360.component.flow360_params.volume_zones import (
+    FluidDynamicsVolumeZone,
+    HeatTransferVolumeZone,
+    InitialConditionHeatTransfer,
+)
+from flow360.exceptions import Flow360ValidationError
+from tests.utils import compare_to_ref, to_file_from_file_test
+
+assertions = unittest.TestCase("__init__")
+
+
+@pytest.fixture(autouse=True)
+def change_test_dir(request, monkeypatch):
+    monkeypatch.chdir(request.fspath.dirname)
+
+
+def test_bet_disks():
```

Comment:
lets add some tests also when using files, for example:
```python
param = Flow360Params('case_with_incorrect_bet.json')
```
and see if it throws the same error.

Reply:
left for a future PR